modules/client-configuration/pages/package-management.adoc

```diff
@@ -47,3 +47,51 @@ When the profile has been created, you can use it to install more clients with t
 . In the {productname} {webui}, navigate to [guimenu]``Systems``, click the client to compare, and navigate to the menu:Systems[Software > Packages > Profiles] subtab.
     To compare with a stored profile, select the profile and click btn:[Compare].
 . To compare with another client, select the client name and click btn:[Compare] to see a list of differences between the two clients.
+
+
+
+== Orphaned Packages
+
+Orphaned packages do not have associated channels.
+Therefore, orphaned packages are usually not available to {productname} clients and cannot install such packages.
+
+Orphaned packages can exist as a result of one of the following events:
+
+
+* A repository removes a package.
+  By default, in {productname} 5.0, we also unlink such package from the channel, but do not remove it.
+  In {productname} 4.3, this was the strict mode behavior.
```

Comment:
Very nitpicky here... :) WDYT about:

- Remove "we"; IMHO there's no we, there's the product as the actor.
- Provide a few clarifications.
- Specify that the `strict mode` was a channel configuration.

```suggestion
* A synchronized repository removes a package.
  By default, {productname} 5.0 unlinks such package from the channel, but does not remove the package.
  In {productname} 4.3, you can enable this behavior by using the `strict mode` channel configuration.
```